src/main/java/me/namuhuchutong/ladder/domain/implement/Row.java

```diff
@@ -56,27 +31,32 @@ private void validateContinuousScaffold(List<LadderExpression> values) {
     }
 
     private boolean filteringVerticalBar(LadderExpression expression) {
-        return expression.equals(HYPHEN);
+        return expression.isNotVerticalBar();
     }
 
+    @Override
     public int move(int startPoint) {
         Movements movement = findDestination(startPoint);
         return movement.move(startPoint);
     }
 
     private Movements findDestination(int startPoint) {
-        LadderExpression left = this.values.get(startPoint - 1);
+        LadderExpression left = findExpression(startPoint - 1);
         LadderExpression right = isEndOfCollection(startPoint) ?
```

Comment:
```suggestion
        int leftPoint = startPoint - 1;
        LadderExpression left = findExpression(leftPoint);
        LadderExpression right = blabla(startPoint);
        
        ...
        
    private LadderExpression blabla(int startPoint) {
        if(isEndOfCollection(startPoint)) {
            return findExpression(startPoint);
        }

        int rightPoint = startPoint + 1;
        return findExpression(rightPoint);
    }
```

3항 연산자는 사람이 쉽게 이해하기 힘든 코드라고 생각해요.
숫자연산에 대해서는 사람이 이해하기 쉽게 의미를 넣어주면 좋을것 같아요.